src/components/admin/hotels/manage-hotels.js

```diff
@@ -1,101 +1,148 @@
 import React from "react";
-
-import HotelIcon from "@material-ui/icons/Hotel";
-import Button from "@material-ui/core/Button";
-
-import SaveHotel from "./save-hotel/save-hotel";
-import { columns, seedData } from "./manage-hotels-constants";
+import {columns} from "./manage-hotels-constants";
 import NoItem from "../../common/no-item";
 import SearchPlugin from "../../common/search-plugin";
-import { DataGrid } from "@material-ui/data-grid";
+import {DataGrid} from "@material-ui/data-grid";
+import {ManageHotelContext} from "../../../store/context";
+import HotelClient from '../../../api/hotel-client';
+import Loading from "../../common/loading";
+import Button from "@material-ui/core/Button";
+import HotelIcon from "@material-ui/icons/Hotel";
+import SaveHotel from './save-hotel/save-hotel';
 
 export default class ManageHotels extends React.Component {
-  constructor(props) {
-    super(props);
+    constructor(props) {
+        super(props);
 
-    this.state = {
-      isSaveHotelModalOpen: false,
-      hotelsRows: [],
-      filteredList: [],
-      searchTerm: "",
-    };
+        this.state = {
+            isSaveHotelModalOpen: false,
+            isLoading: false,
+            searchTerm: "",
+            hotelsRows: [],
+            filteredList: [],
+        };
 
-    this.handleSaveHotelModalToggle = this.handleSaveHotelModalToggle.bind(
-      this
-    );
-  }
+        this.handleSaveHotelModalToggle = this.handleSaveHotelModalToggle.bind(this);
+    }
 
-  componentDidMount() {
-    // getting data from API
-    const hotelsRows = seedData;
+    componentDidMount() {
+        this.setState({
+            isLoading: true,
+        });
 
-    this.setState({
-      hotelsRows: hotelsRows,
-      filteredList: hotelsRows,
-    });
-  }
+        HotelClient.getHotels()
+            .then((result) => {
+                if (result.status === 200) {
```

Comment:
axios response promise rejects if there is a http error(ex. 400), and fulfills if http status is OK(ex. 200). So there is no need to check response status and handle errors by catch(callback)